### PR TITLE
Add tdigest requirement

### LINKED ISSUE #70: tdigest should be added to install_requires of setup.py
tdigest should be added to install_requires of setup.py.
Right now it's impossible to install pybrake from github unless tdigest is already installed.
```
Obtaining pybrake from git+https://github.com/airbrake/pybrake.git#egg=pybrake (from -r requirements.txt (line 22))
  Cloning https://github.com/airbrake/pybrake.git to ./src/pybrake
  Running setup.py (path:/var/app/src/pybrake/setup.py) egg_info for package pybrake
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/var/app/src/pybrake/setup.py", line 3, in <module>
        import pybrake
      File "/var/app/src/pybrake/pybrake/__init__.py", line 1, in <module>
        from .notifier import Notifier
      File "/var/app/src/pybrake/pybrake/notifier.py", line 14, in <module>
        from .routes import RouteStats
      File "/var/app/src/pybrake/pybrake/routes.py", line 7, in <module>
        from tdigest import TDigest
    ImportError: No module named 'tdigest'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/var/app/src/pybrake/setup.py", line 3, in <module>

    import pybrake

  File "/var/app/src/pybrake/pybrake/__init__.py", line 1, in <module>

    from .notifier import Notifier

  File "/var/app/src/pybrake/pybrake/notifier.py", line 14, in <module>

    from .routes import RouteStats

  File "/var/app/src/pybrake/pybrake/routes.py", line 7, in <module>

    from tdigest import TDigest

ImportError: No module named 'tdigest'
```

### PR DESCRIPTION
**Pull request by momyc**
Fixes #70

### CODE REVIEW

**Comment by momyc** [comment]
I'm sorry for trying to do it a quick'n'dirty way.

**Comment by momyc** [comment]
@vmihailenco please review at your earliest convenience

File: setup.py [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-def readme():
-  with open('README.md') as f:
+def content_of(file_name):
+  with open(file_name) as f:
     return f.read()
 
 
 setup(name='pybrake',
-      version=pybrake.__version__,
+      version=content_of('pybrake/version.py').strip(),
```
Comment (vmihailenco) [inline setup.py]: Will not this return `version = "0.4.0-beta.4"` instead of `0.4.0-beta.4`?

Comment (momyc) [reply setup.py]: Ah, you're right. Well, `pybrake` requires `tdigest` and that mean you cannot import pybrake from setup.py. I'll reimplement version handling and update the PR.

Comment (momyc) [reply setup.py]: It's been fixed.

**Comment by vmihailenco** [comment]
This looks good - thanks for working on it. Should I merge it?

**Comment by momyc** [comment]
Seems to be working now.

**Comment by vmihailenco** [comment]
Thanks

**Pull request merged**